crates/db_views/src/comment_report_view.rs

```diff
@@ -26,26 +25,21 @@ use lemmy_db_schema::{
     post,
   },
   source::community::CommunityFollower,
-  utils::{
-    actions,
-    actions_alias,
-    functions::coalesce,
-    get_conn,
-    limit_and_offset,
-    DbConn,
-    DbPool,
-    ListFn,
-    Queries,
-    ReadFn,
-  },
+  utils::{actions, actions_alias, functions::coalesce, get_conn, DbPool},
 };
 
-fn queries<'a>() -> Queries<
-  impl ReadFn<'a, CommentReportView, (CommentReportId, PersonId)>,
-  impl ListFn<'a, CommentReportView, (CommentReportQuery, &'a LocalUserView)>,
-> {
-  let all_joins = |query: comment_report::BoxedQuery<'a, Pg>, my_person_id: PersonId| {
-    query
+impl CommentReportView {
```

Comment:
I simplified the specific report views to only *reads* now, not list. The report combined view should always be used for listing reports.

Reply:
Are these read functions really needed? At least lemmy-ui doesnt seem to use them. Would also be possible to move read into `ReportCombinedView`, with a single `/report/read` endpoint which takes `ReportCombinedId`

Reply:
Bump

Reply:
Yes they are needed IMO, the reportombinedid is pretty useless, we could even skip serializing it because it doesn't help at all. 

The main reason its important to keep the specific reads, is that when you ask for a comment report, you might as well get a `CommentReportView` back, rather than having to get multiple types and coerce them. There's no reason to do that.

Reply:
Alright